infra-as-code/bicep/modules/hubNetworking/hubNetworking.bicep

```diff
@@ -111,14 +111,13 @@ param parPrivateDnsZones array = [
   'privatelink.cassandra.cosmos.azure.com'
   'privatelink.gremlin.cosmos.azure.com'
   'privatelink.table.cosmos.azure.com'
-  'privatelink.${parLocation}.batch.azure.com'
+  'privatelink.${toLower(parLocation)}.batch.azure.com'
   'privatelink.postgres.database.azure.com'
   'privatelink.mysql.database.azure.com'
   'privatelink.mariadb.database.azure.com'
   'privatelink.vaultcore.azure.net'
   'privatelink.managedhsm.azure.net'
-  'privatelink.${parLocation}.azmk8s.io'
-  'privatelink.${parLocation}.backup.windowsazure.com'
```

Comment:
Shouldn't this still be in there? It's in the parameters file

Reply:
Backup is removed from here as it gets populated by the var logic in the `privateDnsZone` module. But if someone is using a Parmeter file they still need to specify it